<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackageStatusWindow</class>
 <widget class="QMainWindow" name="PackageStatusWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Check package details/status</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>391</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="checkButton">
       <property name="text">
        <string>Check</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Enter one of the following...</string>
       </property>
       <widget class="QLabel" name="recipientName_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Recipient name</string>
        </property>
       </widget>
       <widget class="QLabel" name="packageCode_label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Package code</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="recipientName">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>40</y>
          <width>241</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="recipientCode">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>241</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
